provider/win_driver_utils.py

```diff
@@ -194,30 +194,133 @@ def copy_file_to_samepath(session, test, params):
                        " the detailed info:\n%s." % output)
 
 
-def memory_leak_check(vm, test, params):
+def enable_driver(vm, test, cmd, device_id):
     """
-    In order to let the driver verifier to catch memory leaks, driver
-    should be unloaded after driver function.Note that if want to use
-    this function, driver verifier should be enabled before driver function.
+    Enable driver.
 
     :param vm: vm object
     :param test: kvm test object
-    :param params: the dict used for parameters
+    :param cmd: Driver enable
+    :param device_id: Device id in windows guest
     """
-    driver_name = params["driver_name"]
-    devcon_path = params["devcon_path"]
-    media_type = params.get("virtio_win_media_type", "iso")
+    session = vm.wait_for_login()
```

Comment:
Could we close this session in the end of this function, in case there are so many sessions opened during test.